### PR TITLE
Update author lists

### PR DESCRIPTION
**Pull request by betatim**
One more update to the author lists in preparation for #1393 

- [x] Is it mergeable?

### CODE REVIEW

File: doc/index.rst [hunk 1]
```diff
@@ -76,21 +76,25 @@ started on khmer development <dev/getting-started>`.
...
-        David Lin, Justin Lippi, Tamer Mansour, Pamela McA'Nulty, Eric
-        McDonald, Jessica Mizzi, Kevin D. Murray, Joshua R. Nahum, Kaben
-        Nanlohy, Alexander Johan Nederbragt, Humberto Ortiz-Zuazaga, Jeramia
-        Ory, Jason Pell, Charles Pepe-Ranney, Zachary N Russ, Erich Schwarz,
-        Camille Scott, Josiah Seaman, Scott Sievert, Jared Simpson, Connor T.
-        Skennerton, James Spencer, Ramakrishnan Srinivasan, Daniel Standage,
-        James A. Stapleton, Joe Stein, Susan R Steinman, Benjamin Taylor, Will
-        Trimble, Heather L. Wiencko, Michael Wright, Brian Wyss, Qingpeng
-        Zhang, en zyme, C. Titus Brown
+:Authors: Daniel Standage, Russell Neches, Hussien F. Alameldin, Ali Aliyari,
```
Comment (betatim) [inline doc/index.rst]: @standage just double checking: having Russel Neches out of alphabetical order is on purpose?

**Comment by standage** [comment]
Ready for review and merge @betatim @luizirber @camillescott @ctb

**Comment by standage** [comment]
Wait.

File: Makefile [hunk 2]
```diff
@@ -356,29 +356,14 @@ convert-release-notes:
 		pandoc --from=markdown --to=rst $${file} > $${file%%.md}.rst; \
 		done
 
-list-authors:
```
Comment (standage) [inline Makefile]: This target was half-baked, and was already getting out of sync with the other two author-related targets.

File: Makefile [hunk 3]
```diff
@@ -356,29 +356,14 @@ convert-release-notes:
...
-		grep -v 'root\|crusoe\|titus'
-	@echo '\author[]{C. Titus Brown}'
-	@echo '\affil[1]{mcrusoe@msu.edu}'
-	@git log --format='\author[]{%aN} \affil[]{%aE}' | sort -uk2 | \
-		awk -F\\ '{print "\\"$$3}' | grep -v \
-		'root\|crusoe\|titus\|waffle\|boyce\|pickett.rodney'
-	# R. Boyce requested to be removed 2015/05/21
-	# via pers correspondence to MRC
-	# P Rodney requested to be removed 2015/06/22 via pers correspondence
-	# to MRC
```
Comment (standage) [inline Makefile]: Moved these personal correspondence notes to `author-skips.txt`.

**Pull request merged**